beesdoo_base/models/partner.py

```diff
@@ -49,19 +51,15 @@ def write(self, values):
                 if command[0] == 2:
                     command[0] = 3
         return super(Partner, self).write(values)
+    
+    @api.multi
+    def _deactivate_active_cards(self):
+        if len(self.member_card_ids) > 0:
```

Comment:
if is useless since the for will not iterate if there is no element but will not crash